docs/INSTALLATION.md

```diff
@@ -80,3 +80,17 @@ production_db_password="xxx"
 ## Editor Support
 
 For enhanced development experience with Elixir, consider installing [`ElixirLS: Elixir support and debugger`](https://marketplace.visualstudio.com/items?itemName=JakeBecker.elixir-ls) from the Visual Studio Marketplace.
+
+### ElixirLS support matrix
```

Comment:
Add a **Recommended versions** to say exactly which versions we use for development

Reply:
Added!